R/volcano_protti.R

```diff
@@ -46,7 +47,7 @@
 #' interactive = TRUE
```

Comment:
You might want to add the facet option to the example so that people know how and when to use it.